cookbooks/aws-parallelcluster-slurm/recipes/init.rb

```diff
@@ -30,7 +30,7 @@
 
   ruby_block "retrieve compute node info" do
     block do
-      slurm_nodename = hit_dynamodb_info
+      slurm_nodename = dynamodb_info
       node.force_default['cluster']['slurm_nodename'] = slurm_nodename
     end
     retries 5
```

Comment:
optional: I'm wondering if it makes sense to drop these retries or reduce it to 2 since AWS CLI should be already retrying in case of failure. Might be worth double checking the default retry for the AWS CLI command we run in dynamodb_info function

Reply:
I've removed the retries here and setup the timeout parameters for the call to the AWS CLI. The AWS CLI will make 3 attempts on timeout errors.